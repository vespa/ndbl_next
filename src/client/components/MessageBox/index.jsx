import React from 'react';
// import css from './style.scss';
import cssMain from '../../../scss/main.scss';

const MessageBox = () => {
  return (
    <React.Fragment>
      <div
        className={`${cssMain.icon} ${cssMain.icon__message}`}
      >
      message
      </div>
    </React.Fragment>
  );
};

export default MessageBox;
