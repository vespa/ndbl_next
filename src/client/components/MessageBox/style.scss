.message-box {
  width:50px;
  margin: 0.8rem 1rem;
  float: right;
  &__counter{
    color: #fff;
  }
}