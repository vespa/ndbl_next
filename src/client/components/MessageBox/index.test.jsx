import React from 'react';
import { shallow } from 'enzyme';
import MessageBox from './index';

describe('[Component: Input]', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<MessageBox />);
  });
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
