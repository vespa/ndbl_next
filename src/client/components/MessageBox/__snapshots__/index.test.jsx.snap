// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component: Input] should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MessageBox />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <div
        className="undefined undefined"
      >
        message
      </div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "message",
        "className": "undefined undefined",
      },
      "ref": null,
      "rendered": "message",
      "type": "div",
    },
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <div
          className="undefined undefined"
        >
          message
        </div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "message",
          "className": "undefined undefined",
        },
        "ref": null,
        "rendered": "message",
        "type": "div",
      },
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
